SpatialGDK/Source/Private/Interop/SpatialPlayerSpawner.cpp

```diff
@@ -56,7 +56,7 @@ void USpatialPlayerSpawner::SendPlayerSpawnRequest()
 
 void USpatialPlayerSpawner::ReceivePlayerSpawnResponse(Worker_CommandResponseOp& Op)
 {
-	UE_LOG(LogTemp, Warning, TEXT("Got a response"));
+	UE_LOG(LogTemp, Warning, TEXT("ReceivePlayerSpawnResponse"));
```

Comment:
I'm deleting this log since it was pretty useless in my other PR.

Reply:
Will delete too.